jdbc-core/src/test/java/com/salesforce/datacloud/jdbc/core/PropertiesTest.java

```diff
@@ -31,35 +33,67 @@
 import org.junit.jupiter.api.Test;
 import salesforce.cdp.hyperdb.v1.HyperServiceGrpc;
 
-class HyperConnectionSettingsTest extends HyperGrpcTestBase {
+class PropertiesTest extends HyperGrpcTestBase {
     private static final String HYPER_SETTING = "querySetting.";
 
     @Test
-    void testGetSettingWithCorrectPrefix() {
+    @SneakyThrows
+    public void testQuerySetting() {
+        val settings = Maps.immutableEntry("querySetting.date_style", "YMD");
+
+        assertWithStatement(
+                statement -> {
+                    val result = statement.executeQuery("SHOW date_style");
+                    result.next();
+                    assertThat(result.getString(1)).isEqualTo("ISO, YMD");
+                },
+                settings);
+    }
+
+    @Test
+    void testGetSettingWithCorrectPrefix() throws DataCloudJDBCException {
```

Comment:
here and other test cases in this file: I am having a hard time to understand the intended test coverage. Can you please rename the test cases / add comments to more clearly articulate the intended test coverage?

Reply:
Thanks! Looks good now